apps/framework-cli/src/cli/routines.rs

```diff
@@ -346,7 +346,8 @@ fn start_leadership_lock_task(redis_client: Arc<Mutex<RedisClient>>, project: Ar
         let mut interval = interval(Duration::from_secs(LEADERSHIP_LOCK_RENEWAL_INTERVAL)); // Adjust the interval as needed
         loop {
             interval.tick().await;
-            if let Err(e) = manage_leadership_lock(&redis_client, &project).await {
+            let cron_registry = CronRegistry::new().await.unwrap();
```

Comment:
Doesn't this create a new CronRegistry on every pass of the loop?

Reply:
omg